### PR TITLE
docs: update

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
fix

**Did you add tests for your changes?**
NA
**If relevant, did you update the documentation?**
yes
**Summary**
improve descriptions.
sync options.
new added `--watch-options-stdin` and `--no-watch-options-stdin`

**Does this PR introduce a breaking change?**
No

**Other information**
https://github.com/webpack/webpack.js.org/pull/4386

### CODE REVIEW

File: packages/webpack-cli/README.md [hunk 1]
```diff
@@ -60,48 +60,49 @@ or
...
-  migrate | m   Migrate a configuration to a new version
-  loader | l    Scaffold a loader repository
-  plugin | p    Scaffold a plugin repository
-  info | i      Outputs information about your system and dependencies
-  serve | s     Run the webpack Dev Server
+  bundle|b [options]                         Run webpack (default command, can be omitted).
+  version|v                                  Output the version number of 'webpack', 'webpack-cli' and 'webpack-dev-server' and commands.
+  help|h                                     Display help for commands and options.
+  serve|s [options]                          Run the webpack dev server.
+  info|i [options]                           Outputs information about your system.
```
Comment (chenxsan) [inline packages/webpack-cli/README.md:70]:
```suggestion
  info|i [options]                           Output information about your system.
```

File: packages/webpack-cli/README.md [hunk 2]
```diff
@@ -60,48 +60,49 @@ or
...
+  -c, --config <value...>    Provide path to a webpack configuration file e.g. ./webpack.config.js.
+  --config-name <value...>   Name of the configuration to use.
+  -m, --merge                Merge two or more configurations using 'webpack-merge'.
+  --env <value...>           Environment passed to the configuration when it is a function.
+  --progress [value]         Print compilation progress during build.
+  -j, --json [value]         Print result as JSON or store it in a file.
+  -d, --devtool <value>      Determine source maps to use.
+  --no-devtool               Do not generate source maps.
+  --entry <value...>         The entry point(s) of your application e.g. ./src/main.js.
+  --mode <value>             Defines the mode to pass to webpack.
```
Comment (chenxsan) [inline packages/webpack-cli/README.md:90]:
```suggestion
  --mode <value>             Define the mode to pass to webpack.
```

**Review by alexander-akait — CHANGES_REQUESTED** [review]
No need, `info` command `outputs ...`, focus on bugs and features

**Review by alexander-akait — CHANGES_REQUESTED** [review]
Again all options and commands should ends on `s`, look at schema in webpack

**Comment by chenxsan** [comment]
> Again all options and commands should ends on `s`, look at schema in webpack

The convention I saw more is not to add `s`, e.g., https://docs.npmjs.com/cli/v6/commands.

Or `man find`:

```
```
image

**Comment by alexander-akait** [comment]
We need discussion about this, because all webpack options uses `s`, I don't want to fix it in only one place, description for webpack options inside webpack core

**Comment by snitin315** [comment]
Yeah, let's keep only docs update and not change descriptions for now.

**Review by evenstensberg — APPROVED** [review]

**Comment by snitin315** [comment]
fixed in master

**Pull request closed without merging**